packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -152,6 +164,35 @@ if (__DEV__) {
     }
     return stack;
   };
+
+  getCurrentServerStackFramesImpl = function(): ReactFrame[] {
```

Comment:
It seems like once you have this, you should be able to rewrite `getCurrentServerStackImpl` on top of it and remove some of the duplication between them.

Reply:
done